chef/cookbooks/cinder/templates/default/cinder.conf.erb

```diff
@@ -1644,7 +1644,7 @@ nfs_shares_config=/etc/cinder/nfs_shares-<%= backend_id %>
 
 # Mount options passed to the nfs client. See section of the nfs man page for
 # details. (string value)
-#nfs_mount_options = <None>
+nfs_mount_options = <%= volume['netapp']['nfs_mount_options'] %>
```

Comment:
Ah, that's a good thing to know (leaving the old value in place) I think that convention was not always followed.